### DISCUSSION TITLE
Routing option for safety

### DISCUSSION THREAD

**Discussion by ghost** [organicmaps/organicmaps#2370] [Ideas]
It would be great if there was an option to avoid unsafe streets and intersections. Even better would be the ability to users to rate areas for safety. I know brouter has an option to avoid unsafe in its routing, but I don't know how it determines this.

**Comment by hb0nd** [comment]
I was thinking about this too, a routing mode that tries to route through lit and main streets, for example highstreets.

**Reply by biodranik** [reply]
What kind of safety do you mean? A criminal one? Safety for a learner-driver? Avoiding high-traffic streets? Now it sounds very vague, while the algorithm should use some concrete values from the OSM database.
